quantum/backlight/backlight_chibios.c

```diff
@@ -150,15 +174,18 @@ void breathing_callback(PWMDriver *pwmp) {
     uint32_t duty                     = cie_lightness(rescale_limit_val(scale_backlight(breathing_table[index] * 256)));
 
     chSysLockFromISR();
-    pwmEnableChannelI(pwmp, BACKLIGHT_PWM_CHANNEL - 1, PWM_FRACTION_TO_WIDTH(&BACKLIGHT_PWM_DRIVER, 0xFFFF, duty));
+    pwmEnableChannelI(&BACKLIGHT_PWM_DRIVER, BACKLIGHT_PWM_CHANNEL - 1, PWM_FRACTION_TO_WIDTH(&BACKLIGHT_PWM_DRIVER, 0xFFFF, duty));
     chSysUnlockFromISR();
 }
 
-// TODO: integrate generic pulse solution
 void breathing_pulse(void) {
```

Comment:
The code that was here was here was mostly intentional, avoiding previous race conditions. This was mostly on the list of things to remove so im not keen on its reintroduction.

Reply:
You mean removing the backlight pules feature as is? I can revert it if you want.

Reply:
I reverted this to the blocking wait.